homeassistant/components/fan/xiaomi_miio_services.yaml

```diff
@@ -0,0 +1,56 @@
+
+set_buzzer_on:
```

Comment:
All platform specific services should be prefixed with the plaform name. So `xiaomi_miio_set_buzzer_on`

Reply:
I'm using a new domain called "airpurifier" at the moment. Should I use "fan" instead and use
fan.xiaomi_miio_set_buzzer_on instead of airpurifier.set_buzzon_on?

Reply:
Domains are owned by components with the same name